freqtrade/exchange/__init__.py

```diff
@@ -91,7 +91,7 @@ def _init_ccxt(self, exchange_config: dict) -> ccxt.Exchange:
                 'secret': exchange_config.get('secret'),
                 'password': exchange_config.get('password'),
                 'uid': exchange_config.get('uid', ''),
-                'enableRateLimit': True,
+                'enableRateLimit': exchange_config.get('ccxt_rate_limit', False),
```

Comment:
I would keep the default as is for now (so unless users "opt-in" to have this disabled, it remains as is) - not sure if having this disabled will create issues before we merge asyncio, but i believe this setting was introduced because of that (see #617).